bin/license-checker-ci

```diff
@@ -88,11 +97,11 @@ function checkLicense(licenseString, packageString) {
 		licenseString = licenseString.substring(0, licenseString.length - 1);
 	} */
 	return (
-		(acceptedSpdxExpr && spdxIds.includes(licenseString) && satisfies(licenseString, acceptedSpdxExpr))
```

Comment:
Just needed to remove `spdxIds.includes(licenseString)` to get support license expressions in `package.json`.